crates/nu-cli/src/completions/completer.rs

```diff
@@ -170,9 +170,9 @@ impl NuCompleter {
                         let new_span = Span::new(flat.0.start, flat.0.end - 1);
 
                         // Parses the prefix. Completion should look up to the cursor position, not after.
-                        let mut prefix = working_set.get_span_contents(flat.0).to_vec();
+                        let mut prefix = working_set.get_span_contents(flat.0);
```

Comment:
Minor style thought, this doesn't need to be a `mut` binding when it is just sliced two lines below.